package.json

```diff
@@ -6,30 +6,39 @@
   "repository": "mozilla/voice-web",
   "main": "server/server.js",
   "dependencies": {
+    "@types/copy-webpack-plugin": "^4.0.1",
+    "@types/extract-text-webpack-plugin": "^3.0.0",
+    "@types/mkdirp": "^0.5.1",
+    "@types/mysql": "0.0.34",
+    "@types/node": "7.0.29",
     "@types/node-fetch": "^1.6.7",
+    "@types/node-static": "^0.7.0",
+    "@types/random-js": "^1.0.30",
     "@types/tape": "^4.2.30",
+    "@types/webpack": "^3.0.13",
+    "@types/webpack-merge": "^4.1.0",
     "aws-sdk": "^2.67.0",
     "bcrypt": "^1.0.2",
+    "bluebird": "^3.5.0",
+    "concurrently": "^3.5.0",
+    "css-loader": "^0.28.7",
+    "cssnano": "^3.10.0",
+    "es6-promise": "^4.1.1",
     "eslint": "^4.7.2",
     "eslint-plugin-prettier": "^2.3.1",
+    "extract-text-webpack-plugin": "^3.0.1",
     "ff": "0.2.1",
     "glob": "7.1.1 ",
-    "gulp": "3.9.1",
-    "gulp-clean": "^0.3.2",
-    "gulp-eslint": "^4.0.0",
-    "gulp-help": "^1.6.1",
-    "gulp-insert": "^0.5.0",
-    "gulp-shell": "0.6.3",
-    "gulp-uglify": "^2.1.2",
     "js-md5": "^0.6.1",
     "mediaserver": "0.1.0",
     "mkdirp": "^0.5.1",
+    "mysql": "2.13.0",
```

Comment:
did we need to add `mysql`? is this for bindings?

Reply:
That was just moved to normal dependencies, didn't check that one if it still needed